build.sh

```diff
@@ -41,8 +84,8 @@ date -u
 # we want the $@ args here to be re-split
 time ${CONTAINER_SUBSYS}  build \
   --build-arg osv4client=${osv4client} \
-  $@ \
-  -t ocm-container .
+  $DOCKER_ARGS \
```

Comment:
we did a good job seperating docekr to the name CONTAINER_SUBSYS, maybe change it to:
```suggestion
  $CONTAINER_ARGS \
```

not sure

Reply:
Ahh, yeah, good point.  I'll make that change.